bokehjs/src/coffee/models/tools/actions/reset_tool.ts

```diff
@@ -25,6 +25,4 @@ ResetTool.prototype.type = "ResetTool"
 
 ResetTool.prototype.default_view = ResetToolView
 
-ResetTool.define({
-  reset_size: [ p.Bool, true ]
-})
+
```

Comment:
Tests will fail due to this line.

Reply:
Oh, whoops.  I'll remove that now.  Thanks!